apps/communications/contacts/js/views/form.js

```diff
@@ -369,6 +371,36 @@ contacts.Form = (function() {
     };
   };
 
+  function showForm(event) {
+    if(event.keyCode == '17') {
```

Comment:
This number is not understandable.
I'd really appreciate it if you could use event.key which more informative.